src/stylesheets/print.scss

```diff
@@ -0,0 +1,29 @@
+@media print {
+  header.usa-header,
+  footer.usa-footer,
+  nav.easi-breadcrumb-nav,
+  .easi-no-print,
+  .skipnav,
+  noscript {
+    display: none;
+  }
+
+  .easi-review-row {
+    display: flex;
```

Comment:
Prince doesn't support CSS grid, so in a few places we will need to supply a flex box or float based alternative.